sale_commitment_date_mandatory/models/sale_order.py

```diff
@@ -0,0 +1,12 @@
+# © 2021 Camptocamp SA
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import fields, models
+
+
+class SaleOrder(models.Model):
+    _inherit = "sale.order"
+
+    commitment_date = fields.Datetime(
+        required=True, default=fields.Datetime.now, copy=True
```

Comment:
How do you request the user to fill the value if there is already a default value?
What about the company security? If you define one, you will plan work in the past.